modules/managers/galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/src/main/java/dev/galasa/selenium/ISeleniumManager.java

```diff
@@ -51,12 +51,6 @@ public interface ISeleniumManager {
      */
     public IWebPage allocateWebPage(String url, IInternetExplorerOptions options) throws SeleniumManagerException;
 
-    /**
-     * Allocate a new WebPage for a provided URL with Opera Options
-     * @throws SeleniumManagerException
-     */
-    public IWebPage allocateWebPage(String url, IOperaOptions options) throws SeleniumManagerException;
```

Comment:
It would've been better to deprecate Opera support first but the last version of the Opera driver is 4.4.0 and we need to bump `selenium-ie-driver` to at least 4.14.0 (which in turn means bumping all the other selenium dependencies to 4.14.0).